mungegithub/mungers/close-stale-pr.go

```diff
@@ -232,11 +231,47 @@ func closePullRequest(obj *github.MungeObject, inactiveFor time.Duration) {
 	obj.ClosePR()
 }
 
+func getInvolvedUsers(obj *github.MungeObject) []string {
+	var users []string
+
+	var assignee string
+	if obj.Issue.Assignee != nil && obj.Issue.Assignee.Login != nil && *obj.Issue.Assignee.Login != "*" {
```

Comment:
Looks like it:
https://github.com/google/go-github/blob/master/github/issues.go#L39

BTW, I assume the issue API works for pulls also?